dmv/test/dmv_test.js

```diff
@@ -41,7 +53,9 @@ describe('DMV', () => {
 
     dmv.enter("Skidding Skye");
     dmv.enter("Drivin' Dave");
-    dmv.nextCustomer();
```

Comment:
I had to step out of class when we went over this. How does the variable 'next' help us make the test better? Are we just naming the object returned from the nextCustomer() method, or is there more to it than that? It looks like this code is adding validation steps between methods, is that correct?